src/plugins/memdump/memdump.cpp

```diff
@@ -428,7 +428,7 @@ bool inspect_stack_ptr(drakvuf_t drakvuf, drakvuf_trap_info_t* info, memdump* pl
     // read up to 512 bytes of stack, this may fail returning a partial result
     // thus, the following for loop analyzes the buffer only up to the `bytes_read` value
 
-    vmi_read(vmi, &ctx, 512, buf, &bytes_read);
+    (void)vmi_read(vmi, &ctx, 512, buf, &bytes_read);
```

Comment:
The real fix would be to check the return value.

Reply:
The comment above indicates that a read error, or partial reading, is normal in this case.

Reply:
Ah yes, I missed that